src/bundle/Resources/views/fieldtypes/edit/ezmedia.html.twig

```diff
@@ -4,7 +4,7 @@
 
 {%- block ezplatform_fieldtype_ezmedia_row -%}
     {% set preview_block_name = 'ezmedia_preview' %}
-    {% set max_file_size = (form.parent.vars.value.fieldDefinition.validatorConfiguration.FileSizeValidator.maxFileSize * 1024 * 1024)|round %}
+    {% set max_file_size = min(form.parent.vars.value.fieldDefinition.validatorConfiguration.FileSizeValidator.maxFileSize * 1024 * 1024, max_upload_size)|round %}
```

Comment:
Missing `(` and `)`, should be as here: https://github.com/ezsystems/ezplatform-admin-ui/pull/368/files#diff-1a697e5a15cc6441ab21b051e917770dR7